tests/Sub/SqsSnsConnectorTest.php

```diff
@@ -23,14 +23,4 @@ public function it_can_use_a_queue_prefix()
 
         $this->assertEquals('https://sqs.eu-west-1.amazonaws.com/13245/default', $queue->getQueue(null));
     }
-
-    /** @test */
-    public function it_can_use_a_queue_suffix()
```

Comment:
why are we removing this?

Reply:
Our `SnsSqsQueue` class, extends the Laravel `SqsQueue` which <= Laravel 6 doesn't support a suffix https://github.com/laravel/framework/blob/6.x/src/Illuminate/Queue/SqsQueue.php